lang/go/type_name_test.go

```diff
@@ -88,19 +88,40 @@ func TestType(t *testing.T) {
 		{"Proto3.string", "string"},
 		{"Proto3.bytes", "[]byte"},
 		{"Proto3.enum", "Proto3_Enum"},
-		{"Proto3.ext_enum", "ptype.Syntax"},
+		{"Proto3.ext_enum", "typepb.Syntax"},
```

Comment:
hmm why did this work before...

Reply:
i'm not actually sure myself on this one, a bit perplexed myself